features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/resources/DefaultImageWireframeHelper.kt

```diff
@@ -240,6 +269,16 @@ internal class DefaultImageWireframeHelper(
         }
     }
 
+    private fun shouldMaskContextualImage(
+        imagePrivacy: ImagePrivacy,
+        usePIIPlaceholder: Boolean,
+        drawable: Drawable,
+        density: Float
+    ): Boolean =
+        imagePrivacy == ImagePrivacy.CONTEXTUAL &&
+            usePIIPlaceholder &&
```

Comment:
shouldn't be an `||` instead of `&&` here ?

Reply:
It should be correct. We only show the "content image" placeholder if we're in contextual masking mode in addition to the other preexisting conditions.